src/brpc/policy/couchbase_protocol.cpp

```diff
@@ -134,16 +134,16 @@ ParseResult ParseCouchbaseMessage(butil::IOBuf* source, Socket* socket,
     msg->meta.append(&local_header, sizeof(local_header));
     source->pop_front(sizeof(*header));
     source->cutn(&msg->meta, total_body_length);
-    if (header->command == CB_BINARY_SASL_AUTH) {
-      if (header->status != 0) {
-        LOG(ERROR) << "Failed to authenticate the couchbase Server.";
-        return MakeParseError(PARSE_ERROR_NO_RESOURCE,
-                              "Fail to authenticate with the couchbase Server");
-      }
-      msg = static_cast<MostCommonMessage*>(socket->release_parsing_context());
-      msg->pi = pi;
-      return MakeMessage(msg);
-    } else {
+    // if (header->command == CB_BINARY_SASL_AUTH) {
+    //   if (header->status != 0) {
+    //     LOG(ERROR) << "Failed to authenticate the couchbase Server.";
+    //     return MakeParseError(PARSE_ERROR_NO_RESOURCE,
+    //                           "Fail to authenticate with the couchbase Server");
+    //   }
+    //   msg = static_cast<MostCommonMessage*>(socket->release_parsing_context());
+    //   msg->pi = pi;
+    //   return MakeMessage(msg);
+    // } else {
```

Comment:
Do you mean to leave these commented lines out?  It's a bit code smelly to have a big block like this commented out.